Lombiq.Tests.UI/OrchardCoreUITestBase.cs

```diff
@@ -218,6 +218,10 @@ public abstract class OrchardCoreUITestBase
     {
         var testManifest = new UITestManifest(_testOutputHelper) { TestAsync = testAsync };
 
+        var originalTestOutputHelper = _testOutputHelper;
+        (_testOutputHelper, var afterTest) =
+            GitHubActionsGroupingTestOutputHelper.CreateWrapper(_testOutputHelper, testManifest);
```

Comment:
Keep in mind that GH Actions is just one CI system that these tests might be run under, so it shouldn't be used all the time. See how `TeamCityMetadataReporter` is used.

Reply:
If you open the method, when `GitHubActionsGroupingTestOutputHelper.IsGitHubEnvironment.Value` is `false` (in other words when `$GITHUB_ENV` is null or empty) this method returns `_testOutputHelper`, so nothing is changed. This way the logic is centralized.